db/migrate/20201011220306_change_start_time_on_parties.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeStartTimeOnParties < ActiveRecord::Migration[5.2]
+  def change
+    change_column :parties, :start_time, :string
```

Comment:
Changed this time column to string instead of time because dealing with testing and party creation with date datatype was pretty tricky. Added a ticket to try and turn this into time / figure out a better way to store as an 'if time' time.